lib/plugin/rtl.js

```diff
@@ -141,9 +141,9 @@ const converterFunctions = {
 					const valueParts = splitBySpace(value);
 					const match = valueParts[0].match(percentagePattern);
 					if (match) {
-						const parsedValue = (match[0].indexOf(".") > -1)
-							? parseFloat(match[0])
-							: parseInt(match[0], 10);
+						const parsedValue = (match[0].indexOf(".") > -1) ?
```

Comment:
IMO this is less readable than before. So I would prefer "before" instead of "after" for the "operator-linebreak" rule.

Reply:
Alternative proposal: I change it to an If/Else construct?

Reply:
Sounds good 👍

Reply:
Done